posthog/models/organization.py

```diff
@@ -79,8 +79,9 @@ class PluginsAccessLevel(models.IntegerChoices):
     updated_at: models.DateTimeField = models.DateTimeField(auto_now=True)
     domain_whitelist: ArrayField = ArrayField(
         models.CharField(max_length=256, blank=False), blank=True, default=list
-    )  # used to allow self-serve account creation based on social login (#5111)
+    )  # Used to allow/block self-serve account creation based on social login (#5111)
     setup_section_2_completed: models.BooleanField = models.BooleanField(default=True)  # Onboarding (#2822)
+    per_project_access: models.BooleanField = models.BooleanField(default=False)
```

Comment:
Suggestion: `project_based_permissioning` (or something around that?)

Reply:
We can always change the user-facing descriptions / naming, agreed it doesn't make change to change internally for now.

Reply:
Alrighty, I actually did end up changing this to `project_based_permissioning` as I had to rework fields anyway to make this completely project-based.